internal/resources/cluster/tkgvsphere/resource_tkgvsphere.go

```diff
@@ -154,6 +154,12 @@ var tkgVsphereNetwork = &schema.Schema{
 	MaxItems:    1,
 	Elem: &schema.Resource{
 		Schema: map[string]*schema.Schema{
+			apiServerPortKey: {
+				Type:        schema.TypeInt,
+				Default:     apiServerPortDefaultValue,
```

Comment:
We should mention about the default value given in the doc also

Reply:
The generated docs contain the default value.